test/thirdparty/emailDelivery.test.js

```diff
@@ -1,432 +0,0 @@
-/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
- *
- * This software is licensed under the Apache License, Version 2.0 (the
- * "License") as published by the Apache Software Foundation.
- *
- * You may not use this file except in compliance with the License. You may
- * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
```

Comment:
We still need these tests since it tests how the third party recipe interacts with the email verification recipe. So you should modify these tests to work with the new structure.

Reply:
The third-party recipe no longer sends any emails, so I'm not sure if these make sense. We should definitely add a test to check that the user is marked as verified if the provider says it should be, but I'm not sure that's here.

Reply:
If the provider says that the user's email is not verified, then it does "ask" the emailverification recipe to send an email. So i guess we should test that part - this will specifically be testing the `getEmailForUserId` function in the thirdparty recipe.

We should also test that the functioning of `getEmailForUserId` in sub recipes (emailpassword and thirdparty) works as expected when the user uses thirdpartyemailpassword. Same goes for thirdpartypasswordless (with thirdparty and passwordless sub recipes)

Reply:
- the `thirdparty` recipe in node never "asks" the to send an email - that happens on the frontend by calling the API of the `emailverification` recipe
- I'll add those tests in a separate PR

Reply:
- One test to add is that you sign up with a provider which has is email verified as true, and then calling the email veirification recipe should say it's already verified.
- One test to add is that you sign up with a provider which has is email verified as false, and then calling the email veirification recipe should send an email.
- One test to add is that you sign up with passwordless (email and phone) and calling email verification should say that that email is already verified.